common/ayon_common/distribution/file_handler.py

```diff
@@ -144,36 +89,38 @@ def download_file_from_google_drive(
 
         os.makedirs(root, exist_ok=True)
 
-        if os.path.isfile(fpath) and RemoteFileHandler.check_integrity(fpath):
-            print(f"Using downloaded and verified file: {fpath}")
-        else:
-            session = requests.Session()
-
-            response = session.get(url, params={"id": file_id}, stream=True)
-            token = RemoteFileHandler._get_confirm_token(response)
-
-            if token:
-                params = {"id": file_id, "confirm": token}
-                response = session.get(url, params=params, stream=True)
-
-            response_content_generator = response.iter_content(32768)
-            first_chunk = None
-            while not first_chunk:  # filter out keep-alive new chunks
-                first_chunk = next(response_content_generator)
-
-            if RemoteFileHandler._quota_exceeded(first_chunk):
-                msg = (
-                    f"The daily quota of the file {filename} is exceeded and "
-                    f"it can't be downloaded. This is a limitation of "
-                    f"Google Drive and can only be overcome by trying "
-                    f"again later."
-                )
-                raise RuntimeError(msg)
-
-            RemoteFileHandler._save_response_content(
-                itertools.chain((first_chunk, ),
-                                response_content_generator), fpath)
-            response.close()
+        # TODO validate checksum of existing file and download
```

Comment:
I take it that for the time being, this just overrides whatever we downloaded, every time, right?

Reply:
Yes.